### PR TITLE
Export variables needed by e2e-runner.sh in the release-1.1 branch.

### PR DESCRIPTION
**Pull request by spxtr**
#19419

I also turned off xtrace as I did previously for e2e.sh in master.

### CODE REVIEW

File: hack/jenkins/e2e.sh [hunk 1]
```diff
@@ -964,12 +966,18 @@ export PATH=${PATH}:/usr/local/go/bin
 export KUBE_SKIP_UPDATE=y
 export KUBE_SKIP_CONFIRMATIONS=y
 
+# Kubemark
+export USE_KUBEMARK="${USE_KUBEMARK:-false}"
```
Comment (ixdy) [inline hack/jenkins/e2e.sh]: Kubemark only exists on master right now, right?

This makes me realize that this practice isn't fully sustainable - if we add more config vars in the future (likely), we shouldn't have to backport default values for those to each of the release branches.

**Comment by spxtr** [comment]
@ixdy Agreed. I guess it's okay to leave them with their defaults for now. In the future I'd like to rework e2e-runner.sh, but this should do for now.

PTAL.

**Comment by ixdy** [comment]
LGTM

@k8s-bot test this please

**Comment by ixdy** [comment]
why is the "close and comment" button so large? :(

**Pull request merged**